controller/backup_controller.go

```diff
@@ -777,34 +777,36 @@ func (bc *BackupController) disableBackupMonitor(backupName string) {
 }
 
 func (bc *BackupController) syncBackupStatusWithSnapshotCreationTimeAndVolumeSize(volume *longhorn.Volume, backup *longhorn.Backup) {
+	bc.logger.Info("Syncing backup status with snapshot creation time and volume size.")
```

Comment:
Do we need this?
If users try to a backupstore contains thousands of backup to a new cluster, this message will flood the log.
You can add a defer function to add a prefix for the error messages.

Reply:
You are right I will add this to every error message instead